### PR TITLE
Move a lot of commands to staging 

### PR DESCRIPTION
**Pull request by soltysh**
This builds on top of https://github.com/openshift/origin/pull/23095 so only last 3 commits matter.

### CODE REVIEW

File: pkg/oc/cli/cli.go [hunk 1]
```diff
@@ -19,49 +19,50 @@ import (
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	ktemplates "k8s.io/kubernetes/pkg/kubectl/util/templates"
 
+	"github.com/openshift/origin/pkg/cmd/flagtypes"
+	"github.com/openshift/origin/pkg/cmd/templates"
+
```
Comment (mfojtik) [inline pkg/oc/cli/cli.go]: nit: remove empty line?

File: staging/src/github.com/openshift/oc/pkg/cli/buildlogs/buildlogs.go [hunk 2]
```diff
@@ -28,7 +29,7 @@ var (
 
 	buildLogsExample = templates.Examples(`
 		# Stream logs from container
-  	%[1]s build-logs 566bed879d2d`)
+	%[1]s build-logs 566bed879d2d`)
```
Comment (mfojtik) [inline staging/src/github.com/openshift/oc/pkg/cli/buildlogs/buildlogs.go]: what changed here?

File: staging/src/github.com/openshift/oc/pkg/cli/rollout/rollout.go [hunk 3]
```diff
@@ -164,7 +164,7 @@ var (
 
 	rolloutStatusExample = templates.Examples(`
 		# Watch the status of the latest rollout
-  	%[1]s rollout status dc/nginx`)
+	%[1]s rollout status dc/nginx`)
```
Comment (mfojtik) [inline staging/src/github.com/openshift/oc/pkg/cli/rollout/rollout.go]: what changed here?

File: staging/src/github.com/openshift/oc/pkg/helpers/cmd/templater.go [hunk 4]
```diff
@@ -0,0 +1,277 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"text/template"
+	"unicode"
+
+	"github.com/openshift/oc/pkg/helpers/term"
```
Comment (mfojtik) [inline staging/src/github.com/openshift/oc/pkg/helpers/cmd/templater.go]: nit: sort this

**Comment by mfojtik** [comment]
few nits, but lgtm overall

/lgtm

File: staging/src/github.com/openshift/oc/pkg/cli/buildlogs/buildlogs.go (hunk 2 above)
Comment (soltysh) [reply staging/src/github.com/openshift/oc/pkg/cli/buildlogs/buildlogs.go]: formatting

File: staging/src/github.com/openshift/oc/pkg/cli/rollout/rollout.go (hunk 3 above)
Comment (soltysh) [reply staging/src/github.com/openshift/oc/pkg/cli/rollout/rollout.go]: ditto

File: staging/src/github.com/openshift/oc/pkg/helpers/cmd/templater.go (hunk 4 above)
Comment (soltysh) [reply staging/src/github.com/openshift/oc/pkg/helpers/cmd/templater.go]: done.

File: pkg/oc/cli/cli.go (hunk 1 above)
Comment (soltysh) [reply pkg/oc/cli/cli.go]: done.

**Comment by soltysh** [comment]
/retest
unit failure not related to this pr

**Pull request merged**